For freebsd, the PRJ_TARGET should have been "FreeBSD", not "freebsd".
But just make it "UNIX" like the others.
--- makefile.conf.orig	2024-09-18 15:17:37 UTC
+++ makefile.conf
@@ -210,11 +210,7 @@ ifeq ($(strip $(findstring darwin, $(TAR
    PRJ_TARGET = Darwin
    OS         = Darwin
 else
-ifeq ($(strip $(findstring freebsd, $(TARGET))),freebsd)
-   PRJ_TARGET = freebsd
-else
    PRJ_TARGET = UNIX
-endif
 OS      = UNIX
 endif
 endif
